src/aspect_ratio.rs

```diff
@@ -1,4 +1,4 @@
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Clone, Copy)]
```

Comment:
Please add here more implementations:
#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq, Eq, Hash, PartialOrd, Ord)]

You might also need to add serde as a cargo dependency.